chain/substreams/proto/codec.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package substreams.entity.v1;
+
+message EntitiesChanges {
+  repeated EntityChange entityChanges = 1;
+}
+
+message EntityChange {
+  string entity = 1;
+  bytes id = 2;
+  uint64 ordinal = 3;
+  enum Operation {
+    UNSET = 0; // Protobuf default should not be used, this is used so that the consume can ensure that the value was actually specified
+    CREATE = 1;
+    UPDATE = 2;
+    DELETE = 3;
+  }
+  Operation operation = 4;
+  repeated Field fields = 5;
+}
+
+message Field {
+  string name = 1;
+  enum Type {
+    UNSET = 0; // Protobuf default should not be used, this is used so that the consume can ensure that the value was actually specified
+    BIGDECIMAL = 1;
+    BIGINT = 2;
+    INT = 3; // int32
+    BYTES = 4;
+    STRING = 5;
+  }
+  Type value_type = 2;
+  bytes new_value = 3;
+  bool new_value_null = 4;
```

Comment:
could we take advantage of optional? https://developers.google.com/protocol-buffers/docs/proto#optional (it has been added to proto3)